src/blueprints/tv2/timeline-object-factories/tv2-empty-graphics-command-timeline-object-factory.ts

```diff
@@ -0,0 +1,42 @@
+import { Tv2GraphicsCommandTimelineObjectFactory } from './interfaces/tv2-graphics-command-timeline-object-factory'
+import { Tv2BlueprintConfiguration } from '../value-objects/tv2-blueprint-configuration'
+import { EmptyTimelineObject } from '../../timeline-state-resolver-types/abstract-types'
+import { DeviceType } from '../../../model/enums/device-type'
+
+export class Tv2EmptyGraphicsCommandTimelineObjectFactory implements Tv2GraphicsCommandTimelineObjectFactory {
```

Comment:
The interface `Tv2GraphicsCommandTimelineObjectFactory` only has, as to my knowlegde, a Viz implementation.
If there is no Viz in the Gallery, then we shouldn't be creating Viz TimelineObjects, so we are creating instance of this class instead.

I guess one could argue that it's fine to just create the Viz TimelineObjects anyway in a Gallery without a Viz since there is no hardware to send it to, so the TimelineObjects wouldn't have an effect?
Any preference for one or the other?

Reply:
It is better that we create empty timeline objects, rather than Viz timeline objects when they aren't needed. So this is fine, but ideally we shouldn't create timeline objects that the setup is not using.